__brick__/.eslintrc.json

```diff
@@ -7,6 +7,11 @@
     "simple-import-sort"
   ],
   "rules": {
+    "@typescript-eslint/strict-boolean-expressions": "off",
+    "@typescript-eslint/no-misused-promises": "off",
+    "@typescript-eslint/explicit-function-return-type": "off",
```

Comment:
Non esplicitare il returnType di una function equivale a scrivere in javascript,  perché stiamo spegnendo questa rule? Qual è il caso d'uso in cui ci rema contro?